docs/developer-docs/latest/development/custom-fields.md

```diff
@@ -176,15 +176,40 @@ The `Input` React component receives several props. The [`ColorPickerInput` file
 
 ### Options
 
-`app.customFields.register()` can pass an additional `options` object with the following parameters:
+Custom fields can have their own settings in the Content-type builder. In
+`app.customFields.register()` we can pass an additional `options` object  with the following parameters enable their use.
 
 | Options parameter | Description                                                                     | Type                    |
 | -------------- | ------------------------------------------------------------------------------- | ----------------------- |
 | `base`         | Settings available in the _Base settings_ tab of the field in the Content-type Builder       | `Object` or  `Array of Objects` |
 | `advanced`     | Settings available in the _Advanced settings_ tab of the field in the Content-type Builder   | `Object` or  `Array of Objects` |
 | `validator`    | Validator function returning an object, used to sanitize input. Uses a [`yup` schema object](https://github.com/jquense/yup/tree/pre-v1).  | `Function`              |
 
-Both `base` and `advanced` settings accept an object or an array of objects, each object being a settings section. Each settings section could include:
+
+By default, Strapi has options available for it's [field model](/developer-docs/latest/backend-customization/models.md#validations) you can use them in your custom field by copying the object value from [this file](https://github.com/strapi/strapi/blob/e033275bd0915049fa4fa823f00c17ad53a8fc44/packages/core/content-type-builder/admin/src/components/FormModal/attributes/attributeOptions.js#L1) and respecting the structure below. 
+
+```javascript
+{
+	// The option name
+  name: 'required', 
+	// The input type for this option
+  type: 'checkbox',
+	// The default value for this option
+	defaultValue: true,
+	// The translation object for this option's label
+  intlLabel: {
+    id: getTrad('form.attribute.settings.default'),
+    defaultMessage: 'Default value',
+  },
+	// The translation object for this option's description
+	intlDescription: {
+    id: getTrad('form.attribute.settings.default.description'),
+    defaultMessage: 'Default value description',
+	}
+}
+```
+
+Should you not want to use the defaults, `options` enables the creation of Custom settings when we set up or edit a custom field in the Content-types Builder. Both `base` and `advanced` settings accept an object or an array of objects, each object being a settings section. Each settings section could include:
 
 - a `sectionTitle` to declare the title of the section as an [`IntlObject`](https://formatjs.io/docs/react-intl/)
 - and a list of `items` as an array of objects.
```

Comment:
I think we need to make an important distinction here because you don't have to provide the sectionTitle if you want everything to be in one section.

We could call what's described in the table below a `settingItem`, base and advanced tabs can have an array of `settingItem`s and they would appear in the top of the form with no section title. 

If you want to organize your settings into sections you would then need an object with a `sectionTitle` key and an `items` key. The `items` key would take an array of `settingItem`s for that section. If you want a new section but no title (ie you just want some settings grouped together). You can set `sectionTitle` to null.

The color-picker is a good example of both of these cases: https://github.com/strapi/strapi/blob/3a33df8e7e53547d82d4fb65e9fa8395a01b8b7b/packages/plugins/color-picker/admin/src/index.js#L31